modular_skyrat/modules/goofsec/code/sec_clothing_overrides.dm

```diff
@@ -1,5 +1,7 @@
 /////////
 // Overwrites all the security icons with our own bluesec versions; this means little to no mapping/spawning conflicts!
+// (Also includes 'old' versions at the bottom of the file, for the purpose of crates/vendors/admin shenanigans. Can't remove them ENTIRELY after all!)
+//PLASMA MEN ARE TO BE LEFT ALONE!!! ARMADYNE/LOPLAND DONT MAKE PEACEKEEPER PLASMASUITS!
```

Comment:
we should just sort out the plasmamen outfits already

Reply:
I'm getting it next PR dont worry ;)